### ISSUE TITLE
Memorization is not as in the paper, ours is very low

### ISSUE THREAD

**Issue by Alfred-N** [YinengWang/Supervised-Image-Classification-with-Noisy-Labels-Using-Deep-Learning#13]
We need to figure out why noisy labels aren't being memorized. Right now the basic model and ELR seem very equal, while in the paper, basic ResNet with CE memorized a lot of noisy samples.

![image](https://user-images.githubusercontent.com/47230525/118334290-8c6ba800-b50d-11eb-932e-d569d1c008a8.png)

Our basic CE model shows no signs of memorization (always <1%). I personally dont know what is wrong, we should look at the authors' code.

**Comment by YinengWang** [comment]
This graph shows the fraction of "sample with wrong labels", but we compute the fraction of correct, incorrect, and memorized samples with respect to all labels, both correct and wrong. A small change in the code should be done. I'll work on it.

**Comment by rlindsberg** [comment]
> This graph shows the fraction of "samples with wrong labels", but we compute the fraction of correct, incorrect, and memorized samples with respect to all labels, both correct and wrong. A small change in the code should be done. I'll work on it.

Agreed!

**Comment by Alfred-N** [comment]
> This graph shows the fraction of "samples with wrong labels", but we compute the fraction of correct, incorrect, and memorized samples with respect to all labels, both correct and wrong. A small change in the code should be done. I'll work on it.

Great

**Comment by YinengWang** [comment]
I reran the experiment with 40% noise. In the graph, the memorization is still low.

![Screen Shot 2021-05-15 at 9 48 27 AM](https://user-images.githubusercontent.com/77743367/118352632-bef3c000-b562-11eb-8499-847e54e9f616.png)

A reasonable explanation is that they compute memorization on the training data, unlike us, on the testing data (you can't memorize something you didn't see before). So I'll attempt to experment on the training data.

**Comment by YinengWang** [comment]
<img width="841" alt="Screen Shot 2021-05-15 at 10 41 35 AM" src="https://user-images.githubusercontent.com/77743367/118354192-29f4c500-b56a-11eb-88b3-cbeeb67506e9.png">

This time the result seems much better. I'll do experiments on different noise levels.

BTW, I'm afraid they don't have any code involving experiments. They only have models and trainers.

**Comment by Alfred-N** [comment]
Yees!! That looks exactly like their plot! Well done. Can you do a PR with this code?
